pay-api/src/pay_api/__init__.py

```diff
@@ -41,6 +41,7 @@
 def create_app(run_mode=os.getenv('FLASK_ENV', 'production')):
     """Return a configured Flask App using the Factory method."""
     app = Flask(__name__)
+    app.env = run_mode
```

Comment:
Had to add this, otherwise in test mode it wouldn't read from the config file.

Reply:
that's weird. Was it not getting any values or just not setting the test config version of them?

Reply:
Oh was it just not getting ff values?

Reply:
I'd run the unit tests in the debugger, would always be set to PRODUCTION strangely